spring-data-neo4j-rx/src/test/java/org/neo4j/springframework/data/integration/imperative/DynamicRelationshipsIT.java

```diff
@@ -116,6 +154,34 @@ public void shouldWriteDynamicRelationships(@Autowired PersonWithRelativesReposi
 		}
 	}
 
+	@Test // GH-216
+	void shouldWriteDynamicCollectionRelationships(@Autowired PersonWithRelativesRepository repository) {
+
+		PersonWithRelatives personWithRelatives = new PersonWithRelatives("Test");
+		Map<String, List<Pet>> pets;
+		pets = personWithRelatives.getPets();
```

Comment:
```suggestion
		Map<String, List<Pet>> pets = personWithRelatives.getPets();
```
Same for reactive. 
In imperative and reactive `DynamicRelationshipsIT` there is unchanged code that has both lines for `relatives`.